### PR TITLE
Feat: Display price chart

### PR DESCRIPTION
**Pull request by 0xkynz**
image

### CODE REVIEW

File: src/utils/date.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import dayjs from 'dayjs';
 
-export const formatReadDate = (date: string) => {
-  return dayjs(date).format('MMMM D, YYYY h:mm A	');
+export const formatReadDate = (date: string | Date) => {
```
Comment (djhahe) [inline src/utils/date.ts]: formatReadableDate

File: src/hooks/queries/useGetMarketNFTFloorPrice.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+};
+
+export type UseGetMarketNFTOptions = UseQueryOptions<
+  number,
+  unknown,
+  number,
+  [string, string | undefined, string | undefined, string]
+>;
+
+export const useGetMarketNFTFloorPrice = (
```
Comment (djhahe) [inline src/hooks/queries/useGetMarketNFTFloorPrice.ts]: I didn't see usage of this hook anywhere ?

Comment (0xkynz) [reply src/hooks/queries/useGetMarketNFTFloorPrice.ts]: Removed it

File: src/utils/date.ts (hunk 1 above)
Comment (0xkynz) [reply src/utils/date.ts]: Renamed

**Review by djhahe — APPROVED** [review]

**Pull request merged**